css/properties/appearance.json

```diff
@@ -36,6 +36,9 @@
               }
             ],
             "firefox": [
+              {
+                "version_added": "80"
```

Comment:
This seems right. Testing https://mdn-bcd-collector.appspot.com/tests/css/properties/appearance in Firefox 79 and 80, it starts returning true in 80.